internal/rollup/rollup_bundle.bzl

```diff
@@ -43,6 +43,7 @@ def write_rollup_config(ctx, plugins=[], root_dir=None, filename="_%s.rollup.con
     root_dir: root directory for module resolution (defaults to None)
     filename: output filename pattern (defaults to `_%s.rollup.conf.js`)
     output_format: passed to rollup output.format option, e.g. "umd"
+    additional_entry_points: additional entry points for code splitting
```

Comment:
how can I learn what are the right values to put here? this is where some expert knowledge of my application is needed, right?

Reply:
yes. the values for `additional_entry_points` should be of the same scope as `entry_point`. for an angular application some expert knowledge is required as the additional entry points are the generated module ngfactory files (from angular-bazel-example):

```
rollup_bundle(
    name = "bundle",
    additional_entry_points = [
        "src/hello-world/hello-world.module.ngfactory",
        "src/todos/todos.module.ngfactory",
    ],
    entry_point = "src/main.prod",
    deps = ["//src"],
)
```

Reply:
> yes. the values for `additional_entry_points` should be of the same scope as `entry_point`. for an angular application some expert knowledge is required as the additional entry points are the generated module ngfactory files (from angular-bazel-example):
> 
> ```
> rollup_bundle(
>     name = "bundle",
>     additional_entry_points = [
>         "src/hello-world/hello-world.module.ngfactory",
>         "src/todos/todos.module.ngfactory",
>     ],
>     entry_point = "src/main.prod",
>     deps = ["//src"],
> )
> ```

Thank for this solution :100: